psi4/src/psi4/libpsio/error.cc

```diff
@@ -46,91 +42,93 @@ namespace psi {
  **
  ** PSIO_ERROR(): Print out an error message for libpsio.
  **
- ** \param unit   = file number
- ** \param errval = error code (defined symbolically, PSIO_ERROR_XXX)
+ ** \param unit     = file number
+ ** \param errval   = error code (defined symbolically, PSIO_ERROR_XXX)
+ ** \param prev_msg = (optional) A previous error message that should be prepended to the text from this function.
  **
  */
-void psio_error(size_t unit, size_t errval) {
-    int i;
-    fprintf(stderr, "PSIO_ERROR: unit = %zu, errval = %zu\n", unit, errval);
+void psio_error(size_t unit, size_t errval, std::string prev_msg /* = ""*/) {
```

Comment:
I had thought some doc scheme picked up on the c-style parameter comments, but I'm not finding evidence. here's a SO on the topic https://stackoverflow.com/a/4989591 . I'm content with having the `/* = ""*/` since (1) it's not a blah.h and blah.cc pair and (2) unlikely to fall out-of-date. But I'm not adamant.

Reply:
FTR, I'm not adamant either, just want dev discussion.